code/__DEFINES/shuttles.dm

```diff
@@ -21,6 +21,7 @@
 #define SHUTTLE_HEIGHT_TOO_LARGE "height_too_large"
 #define SHUTTLE_ALREADY_DOCKED "we_are_already_docked"
 #define SHUTTLE_SOMEONE_ELSE_DOCKED "someone_else_docked"
+#define SHUTTLE_DOCK_OBSTRUCTED "shuttle_dock_obstructed"
```

Comment:
Originally there was a check where shuttles wouldn't dock if the defined dock size was too small, but they would crush everything in the way IF they docked.

@KorPhaeron then removed the the dock zone check for at least the emergency shuttle, back when the Xenobiology lab hadn't been moved 15 tiles to the right on Metastation. And then it still crushed everything, like it would before.